rest-api-spec/src/main/resources/rest-api-spec/test/search/340_doc_values_field.yml

```diff
@@ -1,8 +1,8 @@
 setup:
   - skip:
       features: [ "headers" ]
-      version: " - 2.11.99"
-      reason: "searching with only doc_values was added in 2.12.0"
+      version: " - 2.99.99"
+      reason: "searching with only doc_values was added in 3.0.0"
```

Comment:
I don't think I noticed this on the earlier commits, but this should only be scoped to the DV-only cases. As long as the fields are indexed, the queries will still work on earlier versions.

Reply:
@msfroh we revert these on both main and 2.12 branches so do you think it still matters?

Reply:
@msfroh updated this to only skip for the doc_values case.